conf/modules/align_reads.config

```diff
@@ -52,21 +51,15 @@ process {
         ]
     }
 
-    withName: MINIMAP2_ALIGN_SPLIT {
+    withName: '.*:ALIGN_READS:MINIMAP2_ALIGN_SPLIT' {
         if(params.preset == 'revio' | params.preset == 'pacbio') {
             ext.args = "-y -x map-hifi --secondary=no -Y"
         } else if(params.preset == 'ONT_R9' | params.preset == 'ONT_R10') {
             ext.args = "-y -x map-ont --secondary=no -Y"
         }
```

Comment:
I think you can write something like this (not tested) 
```suggestion
        ext.args = { [
            '-y',
            '--secondary=no',
            '-Y',
            params.preset.matches('revio|pacbio') ? 'map-hifi' : '',
            params.preset.matches('ONT_R9|ONT_R10') ? 'map-ont' : ''
            ].join(' ') }
```
Also nice to use the "long parameter" name whenever possible

Reply:
Nice, will keep the .matches and long parameters in mind.